lbrynet/lbrynet_daemon/auth/client.py

```diff
@@ -46,7 +46,7 @@ def call(self, method, params={}):
         pre_auth_post_data = {
             'version': '1.1',
             'method': method,
-            'params': [params],
+            'params': params,
```

Comment:
This is a change in how the api works. For some reason, the way params got passed in before was as a list with a dict as the first element of the list, and all the params in that dict. That's pretty dumb, and I fixed it when I made the jsonrpc functions explicitly declare their params. It's backwards-compatible for now (see `auth/server.py` lines 198-208) but that will be removed later.